kubel.el

```diff
@@ -442,13 +442,20 @@ Use C-c C-c to kubectl apply the current yaml buffer."
 (defun kubel-apply ()
   "Save the current buffer to a temp file and try to kubectl apply it."
   (interactive)
-  (let ((filename (format "/tmp/kubel/%s-%s.yaml"
-                          (replace-regexp-in-string "\*\\| " "" (buffer-name))
-                          (floor (float-time)))))
-    (unless  (file-exists-p "/tmp/kubel")
-      (make-directory "/tmp/kubel"))
+  (setq dir-prefix (or
```

Comment:
add dir-prefix to filename to save tmp files on remote hosts when 'default-directory' is a tramp dir.